src/mashupDocumentParser.ts

```diff
@@ -6,13 +6,12 @@ import JSZip from "jszip";
 import { section1mPath, defaults, uint8ArrayType, emptyValue, textResultType, formulaSectionNotFoundErr, xmlTextResultType, element, section1PathPrefix, divider, elementAttributes, elementAttributesValues } from "./constants";
 import { arrayUtils } from "./utils";
 import { Metadata } from "./types";
-import { generateSingleQueryMashup } from "./generators";
 import { ArrayReader } from "././utils/arrayUtils"; 
 
 export default class MashupHandler {
```

Comment:
it's not related to your change, but I maybe it would be better that this class would have the same name as this file?